src/plugins/console/server/routes/api/console/proxy/create_handler.ts

```diff
@@ -127,15 +129,22 @@ export const createHandler = ({
       },
     });
   } catch (e: any) {
-    log.error(e);
```

Comment:
Is there a reason why we don't want to log the error? It's good practice to have service logging the errors for analysis or troubleshooting.

Reply:
We are logging error by core->openesarch logger, but not by console logger. Because the error from  will be forwarded to client(browser). No need to log 2 times